Cargo.lock

```diff
@@ -280,12 +551,12 @@ dependencies = [
 name = "vhost-device-i2c"
 version = "0.1.0"
 dependencies = [
- "clap",
+ "clap 3.0.0-beta.2",
```

Comment:
Any reason we need an explicit beta version? Comment?

Reply:
This is from vhost-user-i2c, which [explicitly requests that version](https://github.com/rust-vmm/vhost-device/blob/f42e35e72c5661bb65b3c38356c67d67b69058bd/i2c/Cargo.toml#L15) (presumably because it wants 3.0, but 3.0 wasn't released at the time).

vhost-user-scsi depends on clap v2 (via structopt), so adding it made Cargo rename vhost-device-i2c's clap dependency in the lockfile to disambiguate between the two versions in use.

Reply:
We need to stick with this beta version for now as the next version needs a newer version of Rust (1.54 I think) and the container doesn't support that for now. It is going to be updated soon I heard. Once that is done, we can go to 3.0.7 then.

Reply:
Hey, this repo is now using the latest rust-vmm-ci with Rust 1.54.

Reply:
@Gaelan I moved to a non-beta version of clap now and it doesn't support the yaml stuff anymore, but it has a very simple interface available now. Please look at:

https://github.com/rust-vmm/vhost-device/pull/93